corehq/apps/reports/templates/reports/async/aggregate_user_status.html

```diff
@@ -11,12 +11,12 @@ <h4>{% trans "Users" %}</h4>
               <h2>{{ total_users }}</h2>
           </div>
           <div class="col-sm-4 text-center">
-              <h4>{% trans "Users that have Ever Submitted Data" %}</h4>
-              <h2>{{ submission_series.bucket_series.total_percent }} <small>({{ submission_series.bucket_series.total }})</small></h2>
+              <h4>{% trans "Users who have Ever Submitted Data" %}</h4>
```

Comment:
Is there a reason for this weird capitalization?

Reply:
ha. "important word case" i guess. what do you think would be better?

Reply:
gonna merge but happy to follow up. agree is a little funny though no one complained during UAT.

Reply:
I would have said either Title Case (Users Who Have Ever Submitted Data), or sentence case (Users who have ever submitted data). I guess we don't have convention for this type of thing, but this just struck me as weird. (non-blocking, and probably not worth changing)